CHANGELOG.md

```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
-## [0.14.0] - 2018-04-13
+## [0.14.0] - 2018-05-18
```

Comment:
That's the date of a previous release. Create a `### Added` section bellow unreleased and add the new changes